### ISSUE TITLE
4E4D87EF - Tales of Versperia Trial Version

### ISSUE THREAD

**Issue by Blayer98** [xenia-project/game-compatibility#849]
[Marketplace](https://marketplace.xbox.com/en-GB/Product/Tales-of-Vesperia-Demo/00000000-0000-400d-80df-00014e4d87ef#)

Tested on https://github.com/benvanik/xenia/commit/ae8118d7872676dc0bf89660b7c4200a31a7aab9

# Issues:
Game loads to the logos, after the Namco Tales Studio logo disappears, Xenia will crash.

# Log:
[xenia.zip](https://github.com/xenia-project/game-compatibility/files/1548589/xenia.zip)

# Labels:
(recommended labels)
state:crash
state:load

**Comment by bdeimos** [comment]
This issue is probably obsolete. #101 is more relevant.

**Comment by gibbed** [comment]
@bdeimos No, the Tales of Vesperia and Tales of Vesperia demo are two distinct releases on the marketplace.

**Comment by Etokapa** [comment]
Tested on https://github.com/xenia-project/xenia/commit/f6f524b81

Completed entire demo. 30FPS on 2x resolution with shaders cached.
:no_entry_sign:Do not run with vsync off.
:warning:Minor graphical issues with corrupted textures when entering battles. May be fixed with F5.

![tov (2)](https://user-images.githubusercontent.com/31154606/130357524-3bd6fd11-01c7-48e1-89dc-e0b74b4071a6.png)

<details>
  <summary>More Screenshots</summary>

![tov (1)](https://user-images.githubusercontent.com/31154606/130357522-c06eb610-309d-447c-a933-acbd935136a6.png)
![tov (3)](https://user-images.githubusercontent.com/31154606/130357528-571508a8-6497-4417-80f3-ad3527994e87.png)
![tov (4)](https://user-images.githubusercontent.com/31154606/130357530-01bd6a5c-3fce-43c1-af98-c240b7d2c1ec.png)
![tov (7)](https://user-images.githubusercontent.com/31154606/130357534-9acbb782-c447-4d0a-8eaa-76862a892699.png)
![tov (8)](https://user-images.githubusercontent.com/31154606/130357536-edd91aa1-bbd4-4824-9a33-81be16ccce04.png)
![tov (9)](https://user-images.githubusercontent.com/31154606/130357537-50dc8758-d07f-4e77-9e4a-4ab1fdef48b8.png)
![tov (13)](https://user-images.githubusercontent.com/31154606/130357544-a2118cf4-6942-45c1-998d-a690393c14c7.png)
![tov (15)](https://user-images.githubusercontent.com/31154606/130357552-7b60e1b2-1dd9-48f1-8c8a-56405f4e40c6.png)

</details>

[xenia.zip](https://github.com/xenia-project/game-compatibility/files/7027474/xenia.zip)
[xenia.z01.zip](https://github.com/xenia-project/game-compatibility/files/7027475/xenia.z01.zip)

state-playable | gpu-drawing-corrupt
